helm/app-operator-chart/templates/rbac.yaml

```
@@ -0,0 +1,137 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: {{ .Values.chart }}
```

```suggestion
  name: {{ .Values.name }}-chart
```

I would just add the suffix rather than defining another value. `values.chart` isn't a clear name for me. 

Also can you make the app-operator role the first role here. As it is the most important.